src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/mockservers/PrisonApiMockServer.kt

```diff
@@ -25,19 +25,24 @@ class PrisonApiMockServer(
   @Autowired private val objectMapper: ObjectMapper
 ) : HealthServer(9097, "/health/ping") {
 
-  fun latestMovementsRespondsWith(request: Set<NomsNumber>, responseBody: List<Movement>?) {
-    stubFor(
-      post(urlEqualTo("/api/movements/offenders/?latestOnly=true&movementTypes=ADM"))
-        .withHeader(AUTHORIZATION, equalTo("Bearer $apiClientJwt"))
-        .withHeader(CONTENT_TYPE, equalTo(APPLICATION_JSON_VALUE))
-        .withHeader(ACCEPT, equalTo(APPLICATION_JSON_VALUE))
-        .withRequestBody(equalToJson(objectMapper.writeValueAsString(request)))
-        .willReturn(
-          aResponse()
-            .withHeaders(HttpHeaders(HttpHeader(CONTENT_TYPE, APPLICATION_JSON_VALUE)))
-            .withStatus(OK.value())
-            .withBody(objectMapper.writeValueAsString(responseBody))
-        )
-    )
+  fun latestMovementsRespondsWith(
+    request: Set<NomsNumber>,
+    responseBody: List<Movement>?,
+    withAuthorization: Boolean = true
```

Comment:
How come we need a switch on auth here?

Reply:
As discussed this was needed as the Integration tests weren't setting the authorisation header.